spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/support/SubscriberFactory.java

```diff
@@ -66,8 +66,9 @@ PullRequest createPullRequest(String subscriptionName, Integer maxMessages,
 
 	/**
 	 * Create a {@link SubscriberStub} that is needed to execute {@link PullRequest}s.
+	 * @param subscriptionName the subscription name
 	 * @return the {@link SubscriberStub} used for executing {@link PullRequest}s
 	 */
-	SubscriberStub createSubscriberStub();
+	SubscriberStub createSubscriberStub(String subscriptionName);
```

Comment:
Are we OK with this breaking change?

Reply:
Also here, maybe create an issue to remember to rollback the breaking changes from this PR.